src/V3EmitCBase.h

```diff
@@ -75,11 +75,34 @@ class EmitCBaseVisitor VL_NOT_FINAL : public AstNVisitor {
         return modp == v3Global.rootp()->constPoolp()->modp();
     }
 
+    static bool isAnonOk(const AstVar* varp) {
+        return v3Global.opt.compLimitMembers() != 0  // Enabled
+               && !varp->isStatic()  // Not a static variable
+               && !varp->isSc()  // Aggregates can't be anon
+               && (varp->basicp() && !varp->basicp()->isOpaque());  // Aggregates can't be anon
+    }
+
+    enum class VarClass : uint8_t {
+        Io = 0,
+        Signal = 1,
+        Temporary = 2,
+        Parameter = 3,
+        Other = 4  // E.g.: interface ref
```

Comment:
Use all caps for enum values please.

Reply:
Without the prefixes there is not much use for this so dropped.